packages/fxa-settings/src/components/LinkUsed/index.tsx

```diff
@@ -0,0 +1,48 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { FtlMsg } from 'fxa-react/lib/utils';
+
+type LinkUsedProps = {
+  isForPrimaryEmail: boolean;
+};
+
+const getTemplateValues = (isPrimaryEmail: boolean) => {
+  if (isPrimaryEmail) {
+    return {
+      headerText: 'Primary email already confirmed',
+      headerId: 'primary-email-verification-link-reused',
+    };
+  } else {
+    return {
+      headerText: 'Sign-in already confirmed',
+      headerId: 'signin-verification-link-reused',
+    };
+  }
+};
+
+const LinkUsed = ({ isForPrimaryEmail }: LinkUsedProps) => {
+  // TODO : Metric event(s) for expired link
+
+  const templateValues = getTemplateValues(isForPrimaryEmail);
+
+  return (
+    <>
+      <h1 className="card-header">
+        <FtlMsg id={templateValues.headerId}>
+          {templateValues.headerText}
+        </FtlMsg>
+      </h1>
+
+      <p>
```

Comment:
Adding `className="mt-4 text-sm"` here would standardize the styling with `LinkDamaged` and `LinkExpired` and give the paragraph a bit more breathing room

Reply:
Great idea! I'll fix it!